seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/DefaultFileMaterializer.scala

```diff
@@ -0,0 +1,17 @@
+package org.scalajs.jsenv.selenium
+
+import org.scalajs.core.tools.io.VirtualTextFile
+import org.scalajs.jsenv.VirtualFileMaterializer
+
+/** Materializes virtual files in a temporary directory and links to them
+  * via file://
+  */
+object DefaultFileMaterializer extends FileMaterializer {
+
+  private val materializer = new VirtualFileMaterializer(true)
+
+  override def materialize(vf: VirtualTextFile): Unit = materializer.materialize(vf)
```

Comment:
I think, but needs verification, that it would be better if this method returned a `URL`, and `getAbsolutePath` was removed.